README.md

```diff
@@ -13,7 +15,51 @@ Used together, you get a powerful tool for developing high-performance angular a
 
 This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.
 
-[![rx-angular logo](https://raw.githubusercontent.com/rx-angular/rx-angular/main/docs/images/rx-angular_logo.png)](https://www.rx-angular.io/)
+## Benefits
+
+ - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
+ - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
+ - ✔ Lean and simple: No boilerplate guaranteed
+ - ✔ Well typed and tested
+ - ✔ Backwards compatible: support for Angular > v11
+
```

Comment:
```suggestion
 - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
 - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
 - ✔ Lean and simple: No boilerplate guaranteed
 - ✔ Well typed and tested
 - ✔ Backwards compatible: support for Angular > v11

> **❗ Warning**
> Expect no migration scripts for any change in `experimental` folders
```